samples/contrib/parameterized_tfx_oss/parameterized_tfx_oss.py

```diff
@@ -74,29 +74,38 @@ def _create_test_pipeline(pipeline_root: Text, csv_input_location: Text,
   example_gen = CsvExampleGen(input_base=examples)
   statistics_gen = StatisticsGen(input_data=example_gen.outputs.examples)
   infer_schema = SchemaGen(
-      stats=statistics_gen.outputs.output, infer_feature_shape=False)
+      stats=statistics_gen.outputs.output, infer_feature_shape=False
+  )
   validate_stats = ExampleValidator(
-      stats=statistics_gen.outputs.output, schema=infer_schema.outputs.output)
+      stats=statistics_gen.outputs.output, schema=infer_schema.outputs.output
+  )
   transform = Transform(
       input_data=example_gen.outputs.examples,
       schema=infer_schema.outputs.output,
-      module_file=taxi_module_file)
+      module_file=taxi_module_file
```

Comment:
Nit: I usually add comma to the last line so that more lines can be added without changing existing lines.

Reply:
Thanks. Good idea.